html-block-and-inline/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&reg; Blanket</h1>
+    <h2>by SnuggieStore</h2>
+    <p>Price: <s>$19.99</s> $14.99</p>
```

Comment:
In the future, use `<em>` tags instead of `<i>` tags. It is more semantically correct.